src/test/java/guitests/BoardTests.java

```diff
@@ -277,15 +279,15 @@ public void boards_panelCount_newBoardCreated() {
         PanelControl panelControl = ui.getPanelControl();
 
         traverseMenu("Boards", "New");
-        waitUntilNodeAppears(hasText("OK"));
-        clickOn("OK");
+        waitUntilNodeAppears(hasText("No"));
+        clickOn("No");
```

Comment:
Sure. Jut put that as a comment for readers to understand.
On Apr 7, 2016 10:40 PM, "Liu Xinan" <notifications@github.com> wrote:

> In src/test/java/guitests/BoardTests.java
> <https://github.com/HubTurbo/HubTurbo/pull/1485#discussion_r58882551>:
>
> > @@ -277,15 +279,15 @@ public void boards_panelCount_newBoardCreated() {
> >          PanelControl panelControl = ui.getPanelControl();
> >
> >          traverseMenu("Boards", "New");
> > -        waitUntilNodeAppears(hasText("OK"));
> > -        clickOn("OK");
> > +        waitUntilNodeAppears(hasText("No"));
> > +        clickOn("No");
>
> The prompt is actually something like Do you want to save the current
> board before open a new one? instead of Open a new board anyway?. In
> other words, no matter whether Yes or No is chosen, a new board will be
> created. Just the matter of whether the current board is saved.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1485/files/b5b74cc84c64e746cebc06603695e97364ceead6#r58882551>
>